test/unit/init/CliRunner.test.ts

```diff
@@ -1,98 +1,126 @@
-import * as Path from 'path';
-import type { Loader } from 'componentsjs';
+import * as path from 'path';
+import { Loader } from 'componentsjs';
 import { runCli } from '../../../src/init/CliRunner';
 import type { Setup } from '../../../src/init/Setup';
 
-let calledInstantiateFromUrl: boolean;
-let calledRegisterAvailableModuleResources: boolean;
-let throwError: boolean;
-let outsideResolve: () => void;
-let functionToResolve: Promise<unknown>;
+const mainModulePath = path.join(__dirname, '../../../');
 
 const mockSetup = {
-  setup: jest.fn(),
+  setup: jest.fn(async(): Promise<any> => null),
 } as unknown as jest.Mocked<Setup>;
```

Comment:
This is just personal preference, but since it saves a few characters and you care about that, I like doing this as:
`const mockSetup: jest.Mocked<Setup> = { ... } as any;`